use_cases/eluc/prescriptors/heuristics/heuristics.py

```diff
@@ -74,22 +73,6 @@ def _reco_heuristic(self, pct: float, context_df: pd.DataFrame):
         adjusted = adjusted.drop(["scaled_change", "row_sum", "max_change"], axis=1)
         return adjusted
 
-    def save(self, path: Path):
-        """
-        Saves best column and percentage.
-        """
-        with open(path / "config.json", "w", encoding="utf-8") as file:
-            json.dump({"pct": self.pct, "best_col": self.best_col}, file)
-
-    @classmethod
-    def load(cls, path: Path) -> "EvenHeuristic":
-        """
-        Loads best column and percentage.
-        """
-        with open(path / "config.json", "r", encoding="utf-8") as file:
-            config = json.load(file)
-        return cls(config["pct"], config["best_col"])
-
```

Comment:
Removed old saving and loading in the heuristics that was just there to satisfy the abstract methods.